### DISCUSSION TITLE
Is there a way to change keybinds in the actual UFG user interface?

### DISCUSSION THREAD

**Discussion by k0v3rt** [Hydraxous/UltraFunGuns#25] [Q&A] [answered]
Am I extremely blind and missing something simple

**Comment by Hydraxous** [comment, marked as answer]
You aren't missing anything, unfortunately I am unable to implement a solution to rebinding in-game. To change your binds you can edit the mod configuration at .../ULTRAKILL/BepInEx/config/hydraxous.ULTRAKILL.UltraFunGuns.cfg
